fakeserver/fakeserver_test.go

```diff
@@ -0,0 +1,122 @@
+package fakeserver
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/Betterment/testtrack-cli/fakeassignments"
+
+	"encoding/json"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMain(m *testing.M) {
+	current, exists := os.LookupEnv("TESTTRACK_FAKE_SERVER_CONFIG_DIR")
+	os.Setenv("TESTTRACK_FAKE_SERVER_CONFIG_DIR", "testdata")
```

Comment:
Thoughts on using `ioutil.TempDir` to give as a completely disposable location for this?